packages/app/src/cli/utilities/developer-platform-client.ts

```diff
@@ -163,6 +166,12 @@ export type AppDeployOptions = AppDeployVariables & {
   name: string
 }
 
+export interface DevSessionDeployOptions {
+  shopName: string
```

Comment:
I think I'd switch this everywhere to something like `shopFqdn`, to be clear to the reader that it's not `some-shop`, it's `some-shop.myshopify.com` (or what have you)